src/main/java/testUI/Utils/Logger.java

```diff
@@ -34,6 +35,12 @@ public static void putLogError(String log, Object ...arg) {
         log(LogLevel.ERROR, sf1);
     }
 
+    public static void putSoftAssert(String log, Object ...arg) {
+        String sf1 = formatString(log, arg);
+        log(LogLevel.WARN, sf1);
```

Comment:
I believe this log should be `ERROR` because it is still an error that an assertion failed, even if the test continued its execution. This would also increase traceability in logs.

```suggestion
        log(LogLevel.ERROR, sf1);
```